src/wallet/wallet.ts

```diff
@@ -186,6 +187,13 @@ export const generateBtcWallet = async (testnet: boolean, mnem: string): Promise
         xpub: hdwallet.derive(testnet ? TESTNET_DERIVATION_PATH : BTC_DERIVATION_PATH).toJSON().xpub
     };
 };
+export const generateQtumWallet = async (testnet: boolean, mnem: string): Promise<Wallet> => {
+    const hdwallet = hdkey.fromMasterSeed(await mnemonicToSeed(mnem), testnet ? networks.testnet.bip32 : networks.bitcoin.bip32);
```

Comment:
Should not be here Qtum networks?

Reply:
this is just interface, 
export declare const bitcoin: Network;
export interface Network {
    messagePrefix: string;
    bech32: string;
    bip32: Bip32;
    pubKeyHash: number;
    scriptHash: number;
    wif: number;
}

Reply:
it's not just an interface. the xpub format is based on that values, so there must be QTUM network passed here correctly.